### ISSUE TITLE
Sphinx RTD Theme missing from dev Dockerfile

### ISSUE THREAD

**Issue by kocolosk** [apache/couchdb-docker#58]
I tried to use the existing "dev" Dockerfile but it failed when trying to build the docs because the Sphinx theme was not installed

## Expected Behavior
`docker build dev/` should run to completion

## Current Behavior
```
sphinx-build -b html -a -W -n -A local=1 -D latex_elements.papersize=a4 -d build/doctree src/ build/html
Making output directory...
Running Sphinx v1.2.3

Exception occurred:
  File "conf.py", line 17, in <module>
ImportError: No module named sphinx_rtd_theme
The full traceback has been saved in /tmp/sphinx-err-ORBU8D.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://bitbucket.org/birkenfeld/sphinx/issues/>. Thanks!
Makefile:32: recipe for target 'html' failed
make[1]: *** [html] Error 1
make[1]: Leaving directory '/usr/src/couchdb/src/docs'
Makefile:374: recipe for target 'src/docs/build' failed
make: *** [src/docs/build] Error 2
The command '/bin/sh -c apt-get update -y -qq && apt-get install -y --no-install-recommends     apt-transport-https     build-essential     ca-certificates     curl     default-jdk     erlang-dev     erlang-nox     git     libcurl4-openssl-dev     libicu-dev     libmozjs185-dev     libwxgtk3.0     openssl     pkg-config     python     python-sphinx     texinfo     texlive-base     texlive-fonts-extra     texlive-fonts-recommended     texlive-latex-extra  && curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add -  && echo 'deb https://deb.nodesource.com/node_6.x jessie main' > /etc/apt/sources.list.d/nodesource.list  && echo 'deb-src https://deb.nodesource.com/node_6.x jessie main' >> /etc/apt/sources.list.d/nodesource.list  && apt-get update -y -qq && apt-get install -y nodejs  && npm install -g grunt-cli  && cd /usr/src && git clone https://gitbox.apache.org/repos/asf/couchdb.git  && cd couchdb && git checkout $checkout  && cd /usr/src/couchdb && ./configure && make  && apt-get purge -y     binutils     build-essential     cpp     default-jdk     git     libcurl4-openssl-dev     libicu-dev     libwxgtk3.0     make     nodejs     perl     pkg-config     texinfo     texlive-base     texlive-fonts-extra     texlive-fonts-recommended     texlive-latex-extra  && apt-get autoremove -y  && apt-get install -y libicu52 --no-install-recommends  && rm -rf /var/lib/apt/lists/* /usr/lib/node_modules src/fauxton/node_modules src/**/.git .git' returned a non-zero code: 2
```

## Possible Solution

I tried adding ` python-sphinx-rtd-theme` to the initial `apt-get install` command but then I ended up with this error:

```
sphinx-build -b html -a -W -n -A local=1 -D latex_elements.papersize=a4 -d build/doctree src/ build/html
Making output directory...
Running Sphinx v1.2.3
loading pickled environment... not yet created
building [html]: all source files
updating environment: 112 added, 0 changed, 0 removed
reading sources... [  0%] about
reading sources... [  1%] api/basics
reading sources... [  2%] api/database/bulk-api
reading sources... [  3%] api/database/changes
reading sources... [  4%] api/database/common
reading sources... [  5%] api/database/compact
reading sources... [  6%] api/database/find
reading sources... [  7%] api/database/index
reading sources... [  8%] api/database/misc
reading sources... [  8%] api/database/security
reading sources... [  9%] api/ddoc/common
reading sources... [ 10%] api/ddoc/index
reading sources... [ 11%] api/ddoc/render
reading sources... [ 12%] api/ddoc/rewrites
reading sources... [ 13%] api/ddoc/views
reading sources... [ 14%] api/document/attachments
reading sources... [ 15%] api/document/common
reading sources... [ 16%] api/document/index
reading sources... [ 16%] api/index
reading sources... [ 17%] api/local
reading sources... [ 18%] api/server/authn
reading sources... [ 19%] api/server/common
reading sources... [ 20%] api/server/configuration
reading sources... [ 21%] api/server/index
reading sources... [ 22%] cluster/databases
reading sources... [ 23%] cluster/index
reading sources... [ 24%] cluster/nodes
reading sources... [ 25%] cluster/setup
reading sources... [ 25%] cluster/sharding
reading sources... [ 26%] cluster/theory
reading sources... [ 27%] config/auth
reading sources... [ 28%] config/compaction
reading sources... [ 29%] config/couch-peruser
reading sources... [ 30%] config/couchdb
reading sources... [ 31%] config/externals
reading sources... [ 32%] config/http
reading sources... [ 33%] config/http-handlers
reading sources... [ 33%] config/index
reading sources... [ 34%] config/intro
reading sources... [ 35%] config/logging
reading sources... [ 36%] config/misc
reading sources... [ 37%] config/proxying
reading sources... [ 38%] config/query-servers
reading sources... [ 39%] config/replicator
reading sources... [ 40%] config/services
reading sources... [ 41%] contributing
reading sources... [ 41%] cve/2010-0009
reading sources... [ 42%] cve/2010-2234
reading sources... [ 43%] cve/2010-3854
reading sources... [ 44%] cve/2012-5641
reading sources... [ 45%] cve/2012-5649
reading sources... [ 46%] cve/2012-5650
reading sources... [ 47%] cve/2014-2668
reading sources... [ 48%] cve/index
reading sources... [ 49%] ddocs/ddocs
reading sources... [ 50%] ddocs/index
reading sources... [ 50%] ddocs/views/collation
reading sources... [ 51%] ddocs/views/index
reading sources... [ 52%] ddocs/views/intro
reading sources... [ 53%] ddocs/views/joins
reading sources... [ 54%] ddocs/views/nosql
reading sources... [ 55%] ddocs/views/pagination
reading sources... [ 56%] experimental
reading sources... [ 57%] externals
reading sources... [ 58%] fauxton/index
reading sources... [ 58%] fauxton/install
reading sources... [ 59%] index
reading sources... [ 60%] install/docker
reading sources... [ 61%] install/freebsd
reading sources... [ 62%] install/index
reading sources... [ 63%] install/mac
reading sources... [ 64%] install/setup
reading sources... [ 65%] install/snap
reading sources... [ 66%] install/troubleshooting
reading sources... [ 66%] install/unix
reading sources... [ 67%] install/upgrading
reading sources... [ 68%] install/windows
reading sources... [ 69%] intro/api
reading sources... [ 70%] intro/consistency
reading sources... [ 71%] intro/curl
reading sources... [ 72%] intro/index
reading sources... [ 73%] intro/overview
reading sources... [ 74%] intro/security
reading sources... [ 75%] intro/tour
reading sources... [ 75%] intro/why
reading sources... [ 76%] json-structure
reading sources... [ 77%] maintenance/compaction
reading sources... [ 78%] maintenance/index
reading sources... [ 79%] maintenance/performance
reading sources... [ 80%] query-server/erlang
reading sources... [ 81%] query-server/index
reading sources... [ 82%] query-server/javascript
reading sources... [ 83%] query-server/protocol
reading sources... [ 83%] replication/conflicts
reading sources... [ 84%] replication/index
reading sources... [ 85%] replication/intro
reading sources... [ 86%] replication/protocol
reading sources... [ 87%] replication/replicator
reading sources... [ 88%] whatsnew/0.10
reading sources... [ 89%] whatsnew/0.11
reading sources... [ 90%] whatsnew/0.8
reading sources... [ 91%] whatsnew/0.9
reading sources... [ 91%] whatsnew/1.0
reading sources... [ 92%] whatsnew/1.1
reading sources... [ 93%] whatsnew/1.2
reading sources... [ 94%] whatsnew/1.3
reading sources... [ 95%] whatsnew/1.4
reading sources... [ 96%] whatsnew/1.5
reading sources... [ 97%] whatsnew/1.6
reading sources... [ 98%] whatsnew/2.0
reading sources... [ 99%] whatsnew/2.1
reading sources... [100%] whatsnew/index

looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... 
Theme error:
unsupported theme option 'canonical_url' given
Makefile:32: recipe for target 'html' failed
make[1]: *** [html] Error 1
make[1]: Leaving directory '/usr/src/couchdb/src/docs'
Makefile:374: recipe for target 'src/docs/build' failed
make: *** [src/docs/build] Error 2
```

I didn't debug it further as I'm actually more interested in creating a Dockerfile that builds a minimal Docker image without docs, Fauxton, etc. at the moment.

**Comment by kocolosk** [comment]
I figured that was the issue, the version in Jessie seemed to predate the RtD theme’s changelog ☺️ Thanks for the pointer!

**Comment by wohali** [comment]
Fixed with 45b9dd1cc3ae1449f47cde77481b7f49173a1d9d.
